docs/Concepts/Keypom Protocol/Github Readme/Types of Drops/usage-customization.md

```diff
@@ -16,24 +16,54 @@ pub struct UsageConfig {
     /// When this drop is deleted and it is the owner's *last* drop, automatically withdraw their balance.
     pub auto_withdraw: Option<bool>,
 }
-```
+```  
 
-## `permissions` 
-*Default: `claim` AND `create_account_and_claim`*
+---
+
+## Permissions
+*Default: `claim` AND `create_account_and_claim`*  
+
+The `permissions` parameter assigns which claim methods the user can call with the key you give them. They can either call both, or just one. 
 
 ### Use case
+Pretend you are hosting a talk about your NEAR dApp, Kramerly, and are onboarding a bunch of new user from the audience. You wish to make sure that all these audience member who are onboarded onto your dApp have accounts that take the form `${YOUR_ACCOUNT}.kramerly.near` so that they can receive exclusive rewards when interacting with your dApp.
 
-## `refund_deposit` 
-*Default: False*
+To do this, you can set the `permissions` parameter to `create_account_and_claim` and set the `root_account_id` parameter to `kramerly.near`. This way, you can ensure that everybody must create a new account with the form of `${YOUR_ACCOUNT}.kramerly.near` to interact with your dApp.  
+
+---
+
+## Refund Deposit
+*Default: False*  
+
+This `refund_deposit` parameter defines whether or not a storage deposit for creating a new wallet should be refunded to the drop owner if a new account is not created.
+
+Part of funding a drop is covering the cost for creating a new named account. However, if the `claim` method is called, the deposit will be unused. If left as false, the deposit will be given to the user.
 
 ### Use case
+This parameter is usually a helpful mechanism when onboarding large numbers of users into NEAR. Pretend you are onboarding 100,000 new users onto NEAR. Since they are new, you factor in the storage deposit needed to create all the new wallets into your funding costs. However, it turns out half of the users had existing wallets. 
```

Comment:
```suggestion
This parameter is usually a helpful mechanism when onboarding large numbers of users onto NEAR. Pretend you are onboarding 100,000 new users. Since they are new, you factor in the storage deposit needed to create the new wallets into your funding costs. After the event, however, it turns out half of the users had existing wallets. 
```